pom.xml

```diff
@@ -153,6 +153,8 @@
 
     <!-- Reproducible Builds -->
     <project.build.outputTimestamp>1678513628</project.build.outputTimestamp>
+
+    <allowed.build.jdks>[11,12),[17,18),[20,21),[21,23)</allowed.build.jdks>
```

Comment:
@harawata Did you mean 22,23 in the last spot?

Reply:
@hazendaz ,

I was going to ask you, actually.
I copied the following line from the parent which has `[20,21),[21,22)` and modified it so that the build can run on 22.
https://github.com/mybatis/parent/blob/master/pom.xml#L267

Are you suggesting to use `[11,12),[17,18),[20,21),[22,23)` ?
That would exclude 21 which is LTS version, so it does not seem right.
https://www.oracle.com/java/technologies/java-se-support-roadmap.html